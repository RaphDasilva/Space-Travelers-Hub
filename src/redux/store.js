import { configureStore } from '@reduxjs/toolkit';
import missionReducer from './mission/missionSlice';
import rocketsReducer from './rockets/rockets';

const store = configureStore({
  reducer: {
    mission: missionReducer, rocketsReducer,
  },
});

export default store;
