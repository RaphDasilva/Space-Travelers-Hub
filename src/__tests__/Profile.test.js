import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/store';
import Profiles from '../components/Profile';

it('Test if my_profile renderers correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Profiles />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
