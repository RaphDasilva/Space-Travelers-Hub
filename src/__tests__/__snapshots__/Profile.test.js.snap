// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if my_profile renderers correctly 1`] = `
<div
  className="container"
>
  <div
    className="header"
  >
    <div>
      My Mission
    </div>
    <div
      className="rocketHead"
    >
      My Rocket
    </div>
  </div>
  <div
    className="body"
  >
    <div
      className="mission"
    >
      <div>
        No Reserved Mission
      </div>
    </div>
    <div
      className="rocket"
    >
      <div>
        No Reserved Rockets
      </div>
    </div>
  </div>
</div>
`;
